roles/network/tasks/enable_services.yml

```diff
@@ -1,162 +1,187 @@
 
 - name: Enable named service
-  service: name={{ dns_service }}
-           enabled=yes
+  service:
+    name: "{{ dns_service }}"
+    enabled: yes
   when: named_enabled and named_install
 
 - name: Disable named service
-  service: name={{ dns_service }}
-           enabled=no
+  service:
+    name: "{{ dns_service }}"
+    enabled: no
   when: not named_enabled and named_install
 
 - name: Disable dnsmasq
-  service: name=dnsmasq
-           enabled=no
+  service:
+    name: dnsmasq
+    enabled: no
   when: not dnsmasq_enabled and dnsmasq_install
 
 - name: dnsmasq copy config file to /etc
-  template: src=network/dnsmasq.conf.j2 dest=/etc/
+  template:
+    src: network/dnsmasq.conf.j2
+    dest: /etc/
   when: dnsmasq_enabled and dnsmasq_install
 
 - name: Enable dnsmasq
-  service: name=dnsmasq
-           enabled=yes
+  service:
+    name: dnsmasq
+    enabled: yes
   when: dnsmasq_enabled and dnsmasq_install
 
 - name: Enable DansGuardian
-  service: name=dansguardian
-           enabled=yes
+  service:
+    name: dansguardian
+    enabled: yes
   when: dansguardian_enabled and dansguardian_install
 
 - name: Disable DansGuardian
-  service: name=dansguardian
-           enabled=no
+  service:
+    name: dansguardian
+    enabled: no
   when: not dansguardian_enabled and dansguardian_install
 
 - name: Create xs_httpcache flag
-  lineinfile: dest=/etc/iiab/iiab.env
-              regexp='^HTTPCACHE_ON=*'
-              line='HTTPCACHE_ON=True'
-              state=present
+  lineinfile:
+    dest: /etc/iiab/iiab.env
+    regexp: '^HTTPCACHE_ON=*'
+    line: 'HTTPCACHE_ON=True'
+    state: present
   when: squid_enabled and squid_install
 
 - name: Enable Squid service
-  service: name={{ proxy }}
-           enabled=yes
+  service:
+    name: "{{ proxy }}"
+    enabled: yes
   when: squid_enabled and squid_install
 
 - name: Copy init script and config file
-  template: src={{ item.src }}
-            dest={{ item.dest }}
-            owner={{ item.owner }}
-            group={{ item.group }}
-            mode={{ item.mode }}
+  template:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+    owner: "{{ item.owner }}"
+    group: "{{ item.group }}"
+    mode: "{{ item.mode }}"
   with_items:
-    - src: 'squid/squid-iiab.conf.j2'
-      dest: '/etc/{{ proxy }}/squid-iiab.conf'
-      owner: '{{ proxy_user }}'
-      group: '{{ proxy_user }}'
-      mode: '0644'
+  - src: 'roles/squid/templates/squid-iiab.conf.j2'
+    dest: '/etc/{{ proxy }}/squid-iiab.conf'
+    owner: "{{ proxy_user }}"
+    group: "{{ proxy_user }}"
+    mode: '0644'
   when: squid_enabled and squid_install
 
 - name: Point to Squid config file from startup file
-  lineinfile: regexp='^CONFIG'
-              line='CONFIG=/etc/{{ proxy }}/squid-iiab.conf'
-              dest=/etc/init.d/{{ proxy }}
+  lineinfile:
+    regexp: '^CONFIG'
+    line: 'CONFIG=/etc/{{ proxy }}/squid-iiab.conf'
+    dest: /etc/init.d/{{ proxy }}
```

Comment:
1) roles/network/tasks/enable_services.yml should prob have double-quotes here: `dest: "/etc/init.d/{{ proxy }}"`
2) `tags:` and `with_items:` are generally followed by lines with 2 leading spaces (in the documentation I've seen anyway!?)